templates/add.jinja2

```diff
@@ -35,12 +35,35 @@
                     <option>{{ tag.tag_name }}</option>
                  {% endfor %}
             </datalist>
+        <label>
+            <input type="checkbox" name="schedule_enabled" id="schedule_enabled"
+                   onchange="toggleScheduleOptions(this.checked)" />
+            Schedule this scan periodically
+        </label>
+        
+        <div id="schedule-options" style="display: none;">
+            <label for="schedule_interval">Interval (in minutes):</label>
```

Comment:
I think the user shouldn't see the posssibility to schedule using minutes interval as the standard scans take hours or days, what do you think about changing the field e.g. to a dropdown with possible options starting with 1 day or even a field to provide the number of days?